service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToStartTest.java

```diff
@@ -146,6 +125,8 @@ void shouldNotDisplayAmountToPayFieldWhenCaseIsInReturnedState() {
 
     @Test
     void shouldHaveDraftApplicationDocumentInResponse() {
+        given(feeService.getFeesDataForOrders(any())).willThrow(new FeeRegisterException(300, "duplicate", null));
```

Comment:
i would rather use some positive scenario here, ideally we dont need both shouldHaveDraftApplicationDocumentInResponse and shouldAddAmountToPayFieldToAnOpenedCase
it is not unit test, but i understand it is not part of your story